src/Public/LogRhythm/Case/Get-LrCasePlaybooks.ps1

```diff
@@ -0,0 +1,140 @@
+
+        If a match is not found, this cmdlet will throw exception
+        [System.Collections.Generic.KeyNotFoundException]
+    .EXAMPLE
+        PS C:\> Get-LrCasePlaybooks -Credential $Token -Id 8703
+        ---
+        id                 : E560822B-3685-48DE-AC25-0314B1C4124F
+        name               : Phishing
+        description        : Use this Playbook when someone has received a malicious phishing email that contains malicious code, a link to malicious code, or is employing social engineering to
+                            obtain user credentials.
+
+        originalPlaybookId : 510C7D5B-F058-4748-A948-233FAECB8348
+        dateAdded          : 2019-12-23T13:31:26.0410191Z
+        dateUpdated        : 2019-12-23T13:37:08.0763176Z
+        lastUpdatedBy      : @{number=227; name=Domo, Derby; disabled=False}
+        pinned             : False
+        datePinned         :
+        procedures         : @{total=7; notCompleted=7; completed=0; skipped=0; pastDue=0}
+
+        id                 : 4CAB940D-CFF7-442E-A54A-5D4949FA783D
+        name               : Compromised Account
+        description        : This playbook assists analysts in handling expected cases of a compromised account.
+        originalPlaybookId : 5CD58351-503E-41E4-B36C-F9C29BDD1508
+        dateAdded          : 2019-12-23T13:36:04.3544575Z
+        dateUpdated        : 2019-12-23T13:37:12.0184697Z
+        lastUpdatedBy      : @{number=227; name=Domo, Derby; disabled=False}
+        pinned             : False
+        datePinned         :
+        procedures         : @{total=6; notCompleted=6; completed=0; skipped=0; pastDue=0}
+    .NOTES
+        LogRhythm-API
+    .LINK
+        https://github.com/SmartResponse-Framework/SmartResponse.Framework        
+    #>
+
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false, Position = 0)]
+        [ValidateNotNull()]
+        [pscredential] $Credential = $SrfPreferences.LrDeployment.LrApiToken,
+
+
+        [Parameter(
+            Mandatory = $true,
+            ValueFromPipeline = $true,
+            Position = 1
+        )]
+        [ValidateNotNullOrEmpty()]
+        [object] $Id
+    )
+
+    Begin {
+        $Me = $MyInvocation.MyCommand.Name
+        
+        $BaseUrl = $SrfPreferences.LRDeployment.CaseApiBaseUrl
+        $Token = $Credential.GetNetworkCredential().Password
+
+        # Enable self-signed certificates and Tls1.2
+        Enable-TrustAllCertsPolicy        
+    }
+
+
+    Process {
+        # Get Case Id
+        $IdInfo = Test-LrCaseIdFormat $Id
+        if (! $IdInfo.IsValid) {
+            throw [ArgumentException] "Parameter [Id] should be an RFC 4122 formatted string or an integer."
+        }
+
+        
+        # Request Headers
+        $Headers = [Dictionary[string,string]]::new()
+        $Headers.Add("Authorization", "Bearer $Token")
+        
+        # Retrieve Case Guid
+        $CaseGuid = (Get-LrCaseById -Id $Id).id
```

Comment:
This logic needs to stay, but I've since updated it to be more appropriate.  For this function the CaseGuid needs to be the Case GUID value.  If the Id parameter is passed as an Integer then we need to convert.

`        $IdInfo = Test-LrCaseIdFormat $Id
        if (! $IdInfo.IsValid) {
            throw [ArgumentException] "Parameter [Id] should be an RFC 4122 formatted string or an integer."
        } else {
            # Convert CaseID Into to Guid
            if ($IdInfo.IsGuid -eq $false) {
                # Retrieve Case Guid
                $CaseGuid = (Get-LrCaseById -Id $Id).id
            } else {
                # Id is valid Guid
                $CaseGuid = $Id
            }
        }`